jellyfin_alexa_skill/alexa/setup/manifest/manifest.json

```diff
@@ -44,6 +45,16 @@
             "Alexa sage Jellyfin Player spiele meine Playlist 'Beste Musik'"
           ],
           "description": "Spiele Medien von deinem Jellyfin Server.\n\nQuellcode: https://github.com/infinityofspace/jellyfin_alexa_skill"
+        },
+        "it-IT": {
+          "name": "Jellyfin Player",
+          "summary": "Riproduci media dalla tuo Server Jellyfin",
+          "examplePhrases": [
+            "Alexa chiedi a Jellyfin di riprodurre la mia musica",
```

Comment:
We are currently replace the invocation name in the example phrases with the custom invocation name from the config file. So we expect the name in the example phrases is the same name as the default display name which is currently `Jellyfin Player`. As it now turns out, the current approach is not ideal, as this always requires `Jellyfin Player` as a string in the examples.

**This does not require any changes from your side on this problem.**
This comment is only for tracking and documentation of the problem, so we can adjust the already unreleased new changes accordingly again to this point after this PR is merged.

Reply:
Ok thank you, i will solve the problem asap :)